library/src/main/java/org/chromium/customtabsclient/CustomTabsHelper.java

```diff
@@ -41,10 +41,16 @@
     static final String BETA_PACKAGE = "com.chrome.beta";
     static final String DEV_PACKAGE = "com.chrome.dev";
     static final String LOCAL_PACKAGE = "com.google.android.apps.chrome";
-    private static final String EXTRA_CUSTOM_TABS_KEEP_ALIVE =
-            "android.support.customtabs.extra.KEEP_ALIVE";
-    private static final String ACTION_CUSTOM_TABS_CONNECTION =
-            "android.support.customtabs.action.CustomTabsService";
+
+    @SuppressWarnings("StringBufferReplaceableByString")
+    private static final String EXTRA_CUSTOM_TABS_KEEP_ALIVE = new StringBuilder("android.support")
+            .append(".customtabs.extra.KEEP_ALIVE")
+            .toString();
+
+    @SuppressWarnings("StringBufferReplaceableByString")
+    private static final String ACTION_CUSTOM_TABS_CONNECTION = new StringBuilder("android.support")
+            .append(".customtabs.action.CustomTabsService")
+            .toString();
```

Comment:
This is an ugly hack, but tricks jetifier into not channging this. If this bug is ever fixed, we could and should revert this.